geomstats/geometry/fiber_bundle.py

```diff
@@ -294,7 +311,7 @@ def vertical_projection(self, tangent_vec, base_point):
         """
         caller_name = sys._getframe().f_back.f_code.co_name
         if caller_name == "horizontal_projection":
-            raise NotImplementedError
+            raise NotImplementedError("Vertical projection is not implemented.")
```

Comment:
I read this as if horizontal_projection is the method calling vertical_projection, then we raise the error.

Is this because, otherwise, we have vertical_projection calling horizontal_projection calling vertical_projection, etc... ? Any chance we could use a more friendly syntax?